badger_voter_sdk/collectors/snapshot_collectors.py

```diff
@@ -24,6 +29,79 @@
 """)
 
 
+GET_SINGLE_PROPOSAL_Q = lambda snapshot_id: gql(f"""
+query {{
+  proposals (
+    first: 1,
+    skip: 0,
+    where: {{
+      network_in: ["1"]
```

Comment:
should the networks be paramaterized? thought i read about balancer cross chain voting happening soon, or will it all be via eth snapshot?